src/main/java/org/broadinstitute/hellbender/utils/fermi/FermiLiteAssembler.java

```diff
@@ -15,35 +15,89 @@
  * Class that allows you to exercise Heng Li's fermi-lite assembler.
  * This is backed by JNI native code, which could fail to load.
  */
-public final class FermiLiteAssembler {
+public final class FermiLiteAssembler implements AutoCloseable {
     private static volatile boolean nativeLibLoaded = false;
+    private final ByteBuffer opts;
 
     public FermiLiteAssembler() {
         loadNativeLibrary();
+        opts = createDefaultOptions();
+        opts.order(ByteOrder.nativeOrder()).position(0).limit(opts.capacity());
     }
 
+    @Override
+    public void close() { destroyByteBuffer(opts); }
+
     public interface BasesAndQuals {
         byte[] getBases();
         byte[] getQuals();
     }
 
+    // number of threads; don't use multi-threading for small data sets
+    public int getNThreads() { return opts.getInt(0); }
+    public void setNThreads( final int nThreads ) { opts.putInt(0,nThreads); }
+    // k-mer length for error correction; 0 for auto estimate
+    public int getECKSize() { return opts.getInt(4); }
+    public void setECKSize( final int kSize ) { opts.putInt(4,kSize); }
+    // both occ threshold in ec and tip threshold in cleaning lie in [min_cnt,max_cnt]
+    public int getMinCnt() { return opts.getInt(8); }
+    public void setMinCnt( final int minCnt ) { opts.putInt(8,minCnt); }
+    public int getMaxCnt() { return opts.getInt(12); }
+    public void setMaxCnt( final int maxCnt ) { opts.putInt(12,maxCnt); }
+    // min overlap length during assembly
+    public int getMinAsmOverlap() { return opts.getInt(16); }
+    public void setMinAsmOverlap( final int minAsmOverlap ) { opts.putInt(16,minAsmOverlap); }
+    // during assembly, don't explicitly merge an overlap if shorter than this value
+    public int getMinMergeLen() { return opts.getInt(20); }
+    public void setMinMergeLen( final int minMergeLen ) { opts.putInt(20,minMergeLen); }
+
+    // graph cleaning options -- you'll have to look at the fermi lite code to try to figure out what these do
```

Comment:
ack... That's a scary proposition, but I can imagine that it's the only reasonable explanation for how to understand these parameters.